### ISSUE TITLE
Build errors': nightly.link/OpenMS/OpenMS/workflows/pyopenms-wheels/develop returns 404

### ISSUE THREAD

**Issue by tapaswenipathak** [OpenMS/pyopenms-docs#292]
To reproduce:

`make html` in `docs/` dir.

```
tapaswenipathak@Tapaswenis-MacBook-Pro docs % make html                       
Running Sphinx v4.5.0

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/sphinx/config.py", line 332, in eval_config_file
    exec(code, namespace)
  File "/Users/tapaswenipathak/opensource/OpenMS/pyopenms-docs/docs/source/conf.py", line 58, in <module>
    download_file("https://nightly.link/OpenMS/OpenMS/workflows/pyopenms-wheels/nightly/{0}-wheels.zip?status=completed".format(OS), "wheels.zip")
  File "/Users/tapaswenipathak/opensource/OpenMS/pyopenms-docs/docs/source/conf.py", line 42, in download_file
    with contextlib.closing(urllib.request.urlopen(url,timeout=timeout)) as fp:
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 404: Not Found

make: *** [html] Error 2
tapaswenipathak@Tapaswenis-MacBook-Pro docs % 

```
I think URL:  https://nightly.link/OpenMS/OpenMS/workflows/pyopenms-wheels/nightly/{0}-wheels.zip?status=completed is responding w/ a 404, as nightly.link is reporting a expired link: 

<img width="1532" alt="Screenshot 2022-08-18 at 12 57 01 PM" src="https://user-images.githubusercontent.com/4143778/185335486-e6a6fd7e-48d6-4aa8-aba4-342e93b15a0f.png">

It errors, maybe requires a rebuild or some update/sync? 
https://github.com/oprypin/nightly.link/commit/3eab57bdab6cb7fbfb94cbf87e98b3d8dd826a78

<img width="1106" alt="Screenshot 2022-08-18 at 12 58 49 PM" src="https://user-images.githubusercontent.com/4143778/185336347-0ee6a98b-986c-4515-97b3-ac9ecf8036ab.png">

**Comment by tapaswenipathak** [comment]
Found:

`pyopenms-wheels` is not getting published.

https://github.com/OpenMS/OpenMS/actions/workflows/pyopenms-wheels.yml?page=3
<img width="1532" alt="Screenshot 2022-08-18 at 1 10 12 PM" src="https://user-images.githubusercontent.com/4143778/185338040-da7bea8a-1d41-4019-9e71-30b97729e7fa.png">

Last publish expired 4 months ago:

<img width="1532" alt="Screenshot 2022-08-18 at 1 10 51 PM" src="https://user-images.githubusercontent.com/4143778/185338157-1ee21e36-31e4-46e9-856f-aee06bb29981.png">

**Comment by tapaswenipathak** [comment]
<img width="1532" alt="Screenshot 2022-08-18 at 1 16 20 PM" src="https://user-images.githubusercontent.com/4143778/185339331-070e2b3e-a846-408f-8184-b6c7c0348fef.png">
<img width="1532" alt="Screenshot 2022-08-18 at 1 16 30 PM" src="https://user-images.githubusercontent.com/4143778/185339360-5d33727a-19e6-4cd1-a733-349d2ffd47b8.png">

**Comment by poshul** [comment]
@jpfeuffer I can update the links to go to our pypi site if thats what you mean. I was not aware that we had ever had the nightly.link functionality successfully enabled.

**Comment by jpfeuffer** [comment]
I mean instead of links and some weird get mechanism, just add pyopenms to the requirements.txt (if it can handle multiple source repos).

**Comment by jpfeuffer** [comment]
We currently remove pyopenms from the requirements to install it later with the wheel. See actions.yml
We should use this: https://dock2learn.com/tech/how-to-specify-non-pypi-requirements-in-your-requirements-txt-file/

**Comment by poshul** [comment]
so it looks like we use nightly.link for the build-test as well. And that it 404's regularly there too see https://github.com/OpenMS/pyopenms-docs/actions/runs/3713524133/jobs/6296460023 where it failed twice and succeeded on the third rerun.

**Comment by jpfeuffer** [comment]
Will be fixed with #313
